src/main/java/team4/footwithme/chat/api/request/ChatRequest.java

```diff
@@ -0,0 +1,15 @@
+package team4.footwithme.chat.api.request;
+
+import jakarta.validation.constraints.NotNull;
+import team4.footwithme.chat.service.request.ChatServiceRequest;
+
+public record ChatRequest(
+        @NotNull(message = "채팅방 아이디는 필수입니다.")
+        Long chatroomId,
+        @NotNull(message = "채팅 메세지는 필수입니다.")
+        String message
```

Comment:
```suggestion
        @NotBlank(message = "채팅 메세지는 필수입니다.")
        String message
```

String의 경우 `@NotBlank`로 거시는게 더 좋을 것 같습니다. 메시지에 공백만 들어가는 경우를 막아줄 수 있으니 좋을 것 같아요 !
실제 메신저들에서도 공백을 막고있는지 모르겠지만, 공백만 들어가는 경우가 막혀있다면 `@NotBlank` 공백만쳐도 보낼 수 있다면 `@NotNull` 하면 좋을 것 같아요 !

의도된 동작이라면 `@NotNull`도 괜찮습니다 ☺️

Reply:
카카오톡 같은 경우에는 빈칸이면 전송이 안되더라고요
`@NotBlank` 를 쓰는게 맞는것같습니다. 감사합니다~